scripts/build/ci-build/Dockerfile

```diff
@@ -42,19 +42,18 @@ RUN apt-get update   && \
         xz-utils        \
         lzma-dev
 RUN git clone https://github.com/tpoechtrager/osxcross.git /tmp/osxcross && \
-    cd /tmp/osxcross && git reset --hard $OSX_CROSS_REV && \
-    curl -L ${OSX_SDK_URL}/${OSX_SDK}.tar.xz -o /tmp/osxcross/tarballs/${OSX_SDK}.tar.xz && \
-    ln -s /usr/bin/llvm-dsymutil-6.0 /usr/bin/dsymutil
+      cd /tmp/osxcross && git reset --hard $OSX_CROSS_REV
+RUN curl -fL ${OSX_SDK_URL}/${OSX_SDK}.tar.xz -o /tmp/osxcross/tarballs/${OSX_SDK}.tar.xz
```

Comment:
Nit. Additional `RUN` commands will create new/more layers. Is that intentional?

Reply:
@marefr I'm fairly sure it shouldn't matter before the final stage of the build. In previous stages it should only make things faster (due to layer caching) and easier to debug. Please point out in case I should be wrong :)

Reply:
Oh missed that there was multiple stages. Then it's fine probably.